components/grid/export/pdf.md

```diff
@@ -10,17 +10,223 @@ position: 1
 
 # PDF Export
```

Comment:
We use the component name in article titles for SEO. The articles in the Export section doesn't seem to follow this, so consider updating if you like.